easybuild/easyconfigs/n/ncompress/ncompress-4.2.4.4-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,40 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'ncompress'
+version = '4.2.4.4'
+
+homepage = 'http://ncompress.sourceforge.net/'
+
+description = """
+ Compress is a fast, simple LZW file compressor. Compress does not have the
+ highest compression rate, but it is one of the fastest programs to compress
+ data. Compress is the defacto standard in the UNIX community for compressing
+ files.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+# https://github.com/vapier/ncompress/archive/v4.2.4.4.tar.gz
+source_urls = ['https://github.com/vapier/ncompress/archive/']
+sources = ['%(name)s-%(version)s.tar.gz']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+skipsteps = ['configure']
+
+installopts = """\
+    DESTDIR=%(installdir)s\
+    BINDIR=/bin\
+    MANDIR=/man/man1\
+"""
```

Comment:
Or just collapse to a single line, I see no reason to spread this across multiple lines? In fact, the trailing `\` characters are confusing imho...

```python
installopts = "DESTDIR=%(installdir)s BINDIR=/bin MANDIR=/man/man1"
```